.github/workflows/tests.yml

```diff
@@ -1,38 +1,66 @@
-# This workflow performs static analysis checks.
-name: tests
+# This workflow does static verification of the DafnyCore library
+name: Dafny Core verification
 
 on:
+  # Scheduled to be run sometime after the nightly build of dafny
+  schedule:
+    - cron: "7 10 * * *"
   workflow_dispatch:
-  push:
-    branches: [ master ]
   pull_request:
     branches: [ master ]
 
 jobs:
   verification:
-    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        # nightly-latest to catch anything that breaks these tests in current development
```

Comment:
Definitely happy with the solution of also using a specific nightly build to use the logging! But "nightly-latest" should only be used in the scheduled build, since otherwise the mainline CI can break spontaneously.

Here's how I set it up on the reportgenerator, which I intend to be the template for other projects: https://github.com/dafny-lang/dafny-reportgenerator/pull/17/files You should be able to do something similar, the only difference being that you have to make the matrix variable conditional instead, something like:

```yaml
version: ${{ github.event_name == 'schedule' && ['nightly-latest'] || ['nightly-2023-02-18-ef4f346', '3.11.0'] }}
```

Reply:
OK. But are you comfortable with that tweaking of the test scheduling **after** this PR is merged, so the changes to the script can be properly tested on master-branch files?

Reply:
Sure, it's only going to be a maintenance pain for us if we don't fix it quickly. :)

Reply:
Put in the next PR. I learn new Github action syntax every day.

Reply:
Actually, this syntax does not work with lists -- tried numerous variations. So for now left out the nightly-latest from the CI build and made a separate nightly build. But this cries out for a reusable workflow that I'll figure out later.